examples/progressbar_displayio_blinka.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
```

Comment:
you can set the copyright of this new example to your own name if you are comfortable with that.

Reply:
Not in it for the recognition, but updated it just in case. Will still deflect the blame as necessary should it come up! 😀